mutmut/__main__.py

```diff
@@ -201,17 +201,20 @@ def print_progress(self):
 @click.option('--untested-policy', type=click.Choice(['ignore', 'skipped', 'error', 'failure']), default='ignore')
 @click.option('--pre-mutation')
 @click.option('--post-mutation')
+@click.option('--use-patch-file', help='Only mutate lines added/changed in the given patch file')
```

Comment:
Should group this next to the `@click.option('--use-coverage', is_flag=True, default=False)` line for clarity.